web/yo/app/scripts/directives/addtherapy.js

```diff
@@ -97,6 +98,36 @@ angular.module('oncokbApp')
                     addTherapy(index);
                 };
 
+                function getDrugsInOrder(lowerCaseQuery) {
+                    _.forEach(drugs, function(drug){
```

Comment:
It's minor, but I would rather use _.map to assign the weight. It's more elegant than constantly returning.